apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -84,84 +71,68 @@ class MultipleChoiceOverviewTable extends Component {
     });
   };
 
-  getColumns = (sortable) => {
-    let dataColumns = [
-      {
-        property: 'question',
-        header: {
-          label: commonMsg.question(),
-          props: {
-            style: {
-            ...tableLayoutStyles.headerCell,
-          }},
-          transforms: [sortable],
-        },
-        cell: {
-          props: {
-            style: {
-            ...tableLayoutStyles.cell,
-          }}
-        }
-      },
-      {
-        property: 'percentAnsweredOptionOne',
-        header: {
-          label: commonMsg.answerOptionA(),
-          props: {
-            style: {
-            ...tableLayoutStyles.headerCell,
-            width: 90,
-          }},
-        },
-        cell: {
-          format: answerOptionsOneFormatter,
-          props: {
-            style: {
-            ...tableLayoutStyles.cell,
-            width: 90,
-          }}
-        }
+  getNotAnsweredColumn = (index) => (
```

Comment:
It doesn't look like you use the `index` argument in this function.

Reply:
Good question:  It is being used to map to alphabetMapper to assign a label for each answer column.

Reply:
It's used for that in `getAnswerColumn`, but not in `getNotAnsweredColumn` right?

Reply:
You are right.  Fixed.